contracts/src/v2/Types.sol

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
+pragma solidity 0.8.25;
+
+import {OperatingMode} from "./../types/Common.sol";
+
+// Inbound message from a Polkadot parachain (via BridgeHub)
+struct InboundMessage {
```

Comment:
would suggest to rename as `InboundMessageV2` or error when generate go bindings.

```
 > mage build
2024/10/28 15:53:26 INFO: Skipped ssz generated object: relays/beacon/state/beacon_encoding.go
2024/10/28 15:53:26 INFO: Skipped ssz generated object: relays/beacon/state/beacon_deneb_encoding.go
# github.com/snowfork/snowbridge/relayer/contracts
contracts/gateway.go:47:6: InboundMessage redeclared in this block
	contracts/gateway.go:40:6: other declaration of InboundMessage
Error: running "go build -o build/snowbridge-relay main.go" failed with exit code 1
```

Reply:
https://github.com/Snowfork/snowbridge/pull/1321/commits/af374a65f57ea9ea7d4705f63b78a353fd6a350c FYI